DFC.Digital/DFC.Digital.AutomationTest.Utilities/SpecflowExtensions.cs

```diff
@@ -114,5 +115,10 @@ private static string GetConditionalData(this TableRow item, string field, strin
         {
             return item.ContainsKey(field) ? item[field] : (defaultValue ?? string.Empty);
         }
+
+        private static T GetConditionalData<T>(this TableRow item, string field)
+        {
+            return item.ContainsKey(field) ? (T)(object)item[field] : default(T);
```

Comment:
did you meant to write `return item.ContainsKey(field) ? (T)Convert.ChangeType(item[field], typeof(T)) : default;`

Reply:
Also it makes sense to remove the string only func and change the signature of the new to 
```
private static T GetConditionalData<T>(this TableRow item, string field, T defaultValue = null)
        {
            return item.ContainsKey(field) ? (T)Convert.ChangeType(item[field], typeof(T)) : defaultValue ?? default;
```